streams.json

```diff
@@ -595,10 +764,12 @@
           "repository_type": "GIT",
           "repository_url": "https://github.com/FasterXML/jackson-datatype-jdk8",
           "codebase": "2.8",
-          "tag": "jackson-datatype-jdk8-2.8.9",
+          "tag": "jackson-modules-java8-2.8.9",
+          "group_id": "com.fasterxml.jackson.datatype",
+          "artifact_id": "jackson-datatype-jdk8",
```

Comment:
It looks like it is a duplicated name, in our current streams.json we have "FasterXML-jackson-datatype-jdk8", in prod KB we have "FasterXML-jackson-modules-java8", if there are no objections, I will remove this one inferred from the dictionary to use the one from prod KB instead.